site/src/components/ErrorSummary/ErrorSummary.tsx

```diff
@@ -1,32 +1,112 @@
 import Button from "@material-ui/core/Button"
+import Collapse from "@material-ui/core/Collapse"
+import IconButton from "@material-ui/core/IconButton"
+import Link from "@material-ui/core/Link"
+import { makeStyles, Theme } from "@material-ui/core/styles"
+import CloseIcon from "@material-ui/icons/Close"
 import RefreshIcon from "@material-ui/icons/Refresh"
-import { FC } from "react"
+import { ApiError, getErrorDetail, getErrorMessage } from "../../api/errors"
```

Comment:
```suggestion
import { ApiError, getErrorDetail, getErrorMessage } from "api/errors"
```

Reply:
This failed the chromatic/deploy job apparently. I didn’t retry though, let me try that again.

Reply:
Oh really! Ignore it then. I will look into that on my branch.